### ISSUE TITLE
Accessibility issue with desktop client

### ISSUE THREAD

**Issue by Adriorjalesvidal** [nextcloud/server#22403]
Hi
There is a issue with the Nextcloud Windows client related to accessibility.
If you use a screen reader software to review the screen, you are not able to interact with the main Nextcloud dialog. 
It is a very important issue to visually impaired people like me. 
The buttons are focusable but not poperly labelled, and if you use the mouse to review the dialog the screen reader reads nothing.

Here is the info:

OS: W10 
Screen reader: Non Visual Desktop Access (NVDA)

Steps to reproduce:
1. Install NVDA, it's free and open source
2. Install Nextcloud Windows client
3. Connect to a server. (On this step, accessibility difficulties are present but not critical)
4. Once the program is running, click on the background tasks windows icon from the task bar
5. Click in Nextcloud Icon. The main Nextcloud client dialog appears
6. Try to interact with the screen using NVDA. Nothing will happen, nothing is read, including tabs and mouse focusing.

If you need help with the Windows client development, please tell me how can I do to help you with this application, or how can I join the development team.
Thanks

**Comment by kesselb** [comment]
cc @nextcloud/accessibility

**Comment by tyrylu** [comment]
Agreed. No idea why the designers of the main window decided to use icon only header buttons and because i have no experience with QML, i have honestly no idea whether it it is possible to set accessible names for these (e. g. an equivalent of QWidget::setAccessibleName). There are other issues, for example the indication of selected folders is completely broken in the selective sync dialog and probably others. Honestly, writing a fully accessible QT app is hard, but the settings dialog, for example, is quite good on this front.

**Comment by Adriorjalesvidal** [comment]
If the main dialog window is developed using QT framework, it will be difficult to solve accessibility issues. As I've learned using Linux QT based graphical desktops, screen reader Orca cries when it needs to interact with QT. I think that NVDA supports QT, because I think Virtualbox hypervisor is developed using this languaje, and since it is not the best accessible interface, it is usable. In the other hand, if the settings pages are "accessible" or usable, I don't understand why main dialog is not.
